docs/simreal/simwithreal.rst

```diff
@@ -0,0 +1,62 @@
+**********************************************************************************
```

Comment:
We should probably mention somewhere on this page that `RealRobot` depends on other packages which are not part of the conda environment.

Reply:
ah right, that's very important